taipy/gui_core/viselements.json

```diff
@@ -100,8 +100,21 @@
                     },
                     {
                         "name": "filter",
-                        "type": "str|list[str]",
-                        "doc": "TODO: a list of scenario attributes to filter on."
+                        "type": "bool|str|list[str]",
+                        "default_value": "Config id;Label;Creation date;Cycle label;Cycle start;Cycle end;Primary;Tags",
```

Comment:
Are all attributes listed here? And if yes should this really be the default?

Reply:
from the spec doc